.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,17 @@
+---
```

Comment:
I think is better add here a example to package feature/issue, because is more easily to separe. For example [APP] / [WEB]

Reply:
I don't this is a interesting change right now. I have updated the issue template for bug to follow the Material-UI spec: https://github.com/mui-org/material-ui/blob/master/.github/ISSUE_TEMPLATE/1.bug.md

Later, we can change that to adapt this to our needs. 💃